shell/browser/ui/cocoa/electron_ns_window.mm

```diff
@@ -189,6 +191,38 @@ - (NSTouchBar*)makeTouchBar {
 
 // NSWindow overrides.
 
+- (void)sendEvent:(NSEvent*)event {
+  // Draggable regions only respond to left-click dragging, but the system will
+  // still suppress right-clicks in a draggable region. Forwarding right-clicks
+  // and ctrl+left-clicks allows the underlying views to respond to right-click
+  // to potentially bring up a frame context menu. WebContentsView is now a
+  // sibling view of the NSWindow contentView, so we need to intercept the event
+  // here as NativeWidgetMacNSWindow won't forward it to the WebContentsView
+  // anymore.
+  if (event.type == NSEventTypeRightMouseDown ||
+      (event.type == NSEventTypeLeftMouseDown &&
+       ([event modifierFlags] & NSEventModifierFlagControl))) {
+    // The WebContentsView is added a sibling of BaseWindow's contentView at
+    // index 0 before it in the paint order - see
+    // https://github.com/electron/electron/pull/41256.
+    auto* wcv = shell_->GetContentsView()->children().front().get();
+    if (!wcv)
+      return;
+
+    auto ns_view = static_cast<electron::DelayedNativeViewHost*>(wcv)
+                       ->GetNativeView()
+                       .GetNativeNSView();
+    if (!ns_view)
+      return;
+
+    auto* iwcv = (ElectronInspectableWebContentsView*)ns_view;
```

Comment:
use static_cast here instead of C-style cast

Reply:
done in e802016a51